pkg/kubelet/kuberuntime/labels.go

```diff
@@ -39,6 +40,8 @@ const (
 	containerTerminationMessagePolicyLabel = "io.kubernetes.container.terminationMessagePolicy"
 	containerPreStopHandlerLabel           = "io.kubernetes.container.preStopHandler"
 	containerPortsLabel                    = "io.kubernetes.container.ports"
+
+	windowsHostProcessContainer = "microsoft.com/hostprocess-container"
```

Comment:
is formatting off here?

Reply:
Nope - I think because I separated this new label with a blank line (similar to lines able this snippet) it resets the justrification.

Reply:
Even before my changes the two conts starting with pod were has different spacing.